config/routes.js

```diff
@@ -39,6 +39,11 @@ function generateProxyTargets (host, port, defaultHost, defaultPort) {
       port: port
     },
 
```

Comment:
Should we have a comment here explaining what kind of request we're trying to match? Like the above routes have?

Something like:
`// Match requests to / `

Reply:
Good shout, I'll implement that now